easybuild/easyconfigs/j/jhbuild/jhbuild-3.15.92-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'ConfigureMake'
+
+name = 'jhbuild'
+version = '3.15.92'
+
+homepage = 'https://wiki.gnome.org/action/show/Projects/Jhbuild'
+description = """JHBuild allows you to automatically download and compile “modules” (i.e. source code packages). 
+Modules are listed in “module set” files, which also include dependency information so that JHBuild can discover 
+what modules need to be built and in what order."""
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+
+sources = ['%(version)s.tar.gz']
+source_urls = [('https://github.com/GNOME/jhbuild/archive/')]
+
+# jhbuild is python so it has a python dependency, but we want to give people freedom to use whatever python they
+# chose during a build process
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+builddependencies = [
+    ('binutils', '2.25'),
+]
+
+dependencies = [
+    ('Autotools', '20150215'),
+    ('pkg-config', '0.29.1'),
+    ('gettext', '0.19.8'),
+    ('flex', '2.6.0'),
+    ('Bison', '3.0.4'),
+    ('M4', '1.4.17'),
```

Comment:
Not for `jhbuild`, it is relying on these being present so that it can do it's job (hence the custom sanity check step). `jhbuild` itself will be a build dependency only of anything that uses it for building.